app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ConfirmationsController < ApplicationController
+  def create
+    @donation = find_donation
+
+    @donation.update!(confirmed: true)
+
+    flash[:success] = t(".success")
+
+    redirect_to :back
```

Comment:
Currently, donors can confirm or decline their donations from both `/profile` and `/profile/edit`. 

Do you think that makes `:back` a good idea here, or is there a better solution?